### DISCUSSION TITLE
Add screenshot of the program to the begin of README

### DISCUSSION THREAD

**Discussion by NeizKap** [linuxdeepin/developer-center#7731] [Features Request & Ideas | 特性请求 & 头脑风暴]
I get used to look at screenshot firstly to estimate an appearance of any program before downloading

**Comment by BLumia** [comment]
The deepin apps are mostly preinstalled when using DDE or the deepin distribution, which you unlikely will need a screenshot. IMO adding AppStream metadata and provide the `screenshots` metadata could be more helpful for regular users, so they can preview the app while using an app-store thingy.

Just for curiosity, which repo is the one that lead you here?
